src/main/features/response/routes/when-will-you-pay.ts

```diff
@@ -0,0 +1,60 @@
+import * as express from 'express'
+
+import { Paths } from 'response/paths'
+import { GuardFactory } from 'response/guards/guardFactory'
+
+import { ErrorHandling } from 'common/errorHandling'
+import { DefendantPaymentOption, DefendantPaymentType } from 'response/form/models/defendantPaymentOption'
+import { Form } from 'forms/form'
+import { FormValidator } from 'forms/validation/formValidator'
+import { User } from 'idam/user'
+import { DraftService } from 'services/draftService'
+import { NotFoundError } from '../../../errors'
+
+const accessGuardRequestHandler: express.RequestHandler = GuardFactory.createForFeatureToggle('featureToggles.fullAdmission',
+  (req: express.Request, res: express.Response): void => {
+    throw new NotFoundError(Paths.defencePaymentOptionsPage.uri)
```

Comment:
Do we need to define a not found request handler for every route? I'm quite sure this could be a generic method, path can be taken from the request object.